solutions/fully-configurable/variables.tf

```diff
@@ -42,7 +42,7 @@ variable "region" {
   default     = "us-south"
 }
 
-variable "plan" {
+variable "service_plan" {
```

Comment:
As per other changes like in watsonx ai PR, you mentioned name of service like `watsonx_ai_runtime_plan`, can we make similar change here?